src/components/AppComponent.tsx

```diff
@@ -144,7 +144,7 @@ const AppComponent: FC<Props> = (props) => {
               <Toolbar />
               {tutorial && !isLoading ? <Tutorial /> : null}
               <SplitPane
-                style={{ position: 'relative' }}
+                style={{ position: 'relative', paddingBottom: Math.pow(10, scale * 2.02) }}
```

Comment:
Hi! I think your calculation might be off still. This increases the padding-bottom exponentially, when it should increase proportionally. When I increase the font just a few times, I get thousands of pixels of padding-bottom.

Also, we should try to decouple this from the `SplitPane`.

Reply:
<img width="1440" alt="Screen Shot 2020-04-21 at 9 02 04 PM" src="https://user-images.githubusercontent.com/34966228/79932671-ab376200-8413-11ea-840f-90418d83d72e.png">
<img width="1440" alt="Screen Shot 2020-04-21 at 9 02 16 PM" src="https://user-images.githubusercontent.com/34966228/79932673-ad012580-8413-11ea-9ab8-81ec23b3292f.png">

It doesn't seem to be creating thousands of pixels for me.

Reply:
I'll add some changes

Reply:
`SplitPane` is what's causing the clipping.

Reply:
> `SplitPane` is what's causing the clipping.

Oh, I see.

Reply:
> It doesn't seem to be creating thousands of pixels for me.

Try a font size 10 steps higher. That's still within an order of magnitude so it should support that.

Unless I'm wrong, I believe the height will grow proportionally to the font size, not exponentially.

Reply:
Yeah, I'm going to remove that and do something else.

Reply:
Bottom margin should scale proportionally now. 474d4cd